server/src/main/java/org/opensearch/extensions/UpdateSettingsResponseHandler.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions;
+
+import java.io.IOException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.message.ParameterizedMessage;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.threadpool.ThreadPool;
+import org.opensearch.transport.TransportException;
+import org.opensearch.transport.TransportResponseHandler;
+
+/**
+ * Response handler for {@link UpdateSettingsRequest}
+ *
+ * @opensearch.internal
+ */
+public class UpdateSettingsResponseHandler implements TransportResponseHandler<AcknowledgedResponse> {
+    private static final Logger logger = LogManager.getLogger(UpdateSettingsResponseHandler.class);
+
+    @Override
+    public AcknowledgedResponse read(StreamInput in) throws IOException {
+        return new AcknowledgedResponse(in);
+    }
+
+    @Override
+    public void handleResponse(AcknowledgedResponse response) {
+        logger.info("response {}", response.getStatus());
```

Comment:
We probably should do something with the fact that update settings may fail ....

Reply:
Is that not covered in handleException()?

Reply:
Sorry, how this is covered? `AcknowledgedResponse` could return `acknowledged: false`

Reply:
You're right. I added a check to see if the response is false, and called handleException with a new TransportException inside the if statement